lucas-eugenio/Agreggator
# [FE 0.3.0] Get News From API and Major Refactoring

## What does this PR do?
- Get News from BE.
- Major Refactoring.

## How to test?
- Run Server Locally.
- Add sites to **List**.
- See the **News**.

Page:
![image](https://user-images.githubusercontent.com/39530109/84444085-4cbe8100-ac17-11ea-8598-7625d84d607b.png)

## Critical points
- No critical points.
